src/main/java/com/sktTask/ProductService/repository/ProductRepository

```diff
@@ -0,0 +1,16 @@
+package com.sktTask.ProductService.repository;
+
+import com.sktTask.ProductService.models.Product;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ProductRepository extends JpaRepository<Product, Integer>{
```

Comment:
Don't forget to use Stored Procedures, let us know if you are planning to do it in next PRs, if this already works.